hack/update-helm-index.sh

```diff
@@ -88,8 +88,9 @@ else
 			echo "Skipping $asset_url, does not look like a Helm chart archive"
 			continue
 		fi
-		echo "Downloading $asset_url..."
-		curl -sSfLO -o $HELM_REPO_PATH/stable/$(basename $asset_url) $asset_url
+		target=$HELM_REPO_PATH/stable/$(basename $asset_url)
```

Comment:
This fixes the `update-helm-index`, tested locally.
Thanks for this fix @elezar